R/filters.R

```diff
@@ -234,17 +234,70 @@ get_calibrate_barchart_defaults <- function(filters) {
 }
 
 get_comparison_barchart_defaults <- function(output, filters) {
+  area_default <- get_area_level_filter_option(output)
   list(
-    indicator_id = scalar("prevalence"),
-    x_axis_id = scalar("age"),
-    disaggregate_by_id = scalar("source"),
-    selected_filter_options = list(
-      area = get_area_level_filter_option(output),
-      quarter = get_selected_mappings(filters, "quarter")[2],
-      sex = get_selected_mappings(filters, "sex")[1],
-      age = get_selected_mappings(filters, "age",
-                                  naomi::get_five_year_age_groups()),
-      source = get_selected_mappings(filters, "source")
+    list(
+      indicator_id = scalar("prevalence"),
+      x_axis_id = scalar("age"),
+      disaggregate_by_id = scalar("source"),
```

Comment:
I'd be surprised if these need to vary per indicator? But I guess this way we can keep our options open.

Reply:
Yeah, so `disaggregate_by_id` is special here as we don't show that as a filter for users. But I think possible they might change `x_axis_id`. So I think keeping them configurable makes sense